AnkiDroid/src/main/java/com/ichi2/anki/pages/Statistics.kt

```diff
@@ -37,6 +38,12 @@ class Statistics : PageFragment(R.layout.statistics) {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<AppBarLayout>(R.id.app_bar)
+            .addLiftOnScrollListener { _, backgroundColor ->
+                activity?.window!!.statusBarColor = backgroundColor
```

Comment:
```suggestion
                activity?.window?.statusBarColor = backgroundColor
```

Can we use ? instead of !!